### ISSUE TITLE
C# highlighting in Blazor/Razor @code blocks not working

### ISSUE THREAD

**Issue by egil** [highlightjs/highlightjs-cshtml-razor#14]
It looks as if the highlighting is broken in Blazor/Razor files, where C# code inside `@code`  blocks are not correctly highlighted.

For example, this .razor file:

```cshtml
<Fixture Test="InitialHtmlIsCorrect">
  <ComponentUnderTest>
    <Heading />
  </ComponentUnderTest>
  <Fragment>
    <h3 id:regex="heading-\d{4}" required>
      Heading text
      <small diff:ignore></small>
    </h3>    
  </Fragment>

  @code 
  {
    void InitialHtmlIsCorrect(Fixture fixture)
    {
      // Arrange - Gets the Heading component
      var cut = fixture.GetComponentUnderTest<Heading>();

      // Assert      
      // Here we get expected HTML from the Fragment element above.
      var expectedHtml = fixture.GetFragment();

      // Here we use the HTML diffing library to assert that the rendered HTML
      // from CUT is semantically the same as the expected HTML string above.
      cut.MarkupMatches(expectedHtml);
    }
  }
</Fixture>
```

Becomes this HTML with highlightjs-cshtml-razor:

```html
<code class="lang-razor hljs cshtml-razor" name="SemanticHtmlTest.razor"><span class="hljs-tag">&lt;<span class="hljs-name">Fixture</span> <span class="hljs-attr">Test</span>=<span class="hljs-string">"InitialHtmlIsCorrect"</span>&gt;</span>
  <span class="hljs-tag">&lt;<span class="hljs-name">ComponentUnderTest</span>&gt;</span>
    <span class="hljs-tag">&lt;<span class="hljs-name">Heading</span> /&gt;</span>
  <span class="hljs-tag">&lt;/<span class="hljs-name">ComponentUnderTest</span>&gt;</span>
  <span class="hljs-tag">&lt;<span class="hljs-name">Fragment</span>&gt;</span>
    <span class="hljs-tag">&lt;<span class="hljs-name">h3</span> <span class="hljs-attr">id:regex</span>=<span class="hljs-string">"heading-\d{4}"</span> <span class="hljs-attr">required</span>&gt;</span>
      Heading text
      <span class="hljs-tag">&lt;<span class="hljs-name">small</span> <span class="hljs-attr">diff:ignore</span>&gt;</span><span class="hljs-tag">&lt;/<span class="hljs-name">small</span>&gt;</span>
    <span class="hljs-tag">&lt;/<span class="hljs-name">h3</span>&gt;</span>    
  <span class="hljs-tag">&lt;/<span class="hljs-name">Fragment</span>&gt;</span>

  <span class="hljs-built_in">@</span><span class="csharp">code</span> 
  {
    void InitialHtmlIsCorrect(Fixture fixture)
    {
      // Arrange - Gets the Heading component
      var cut = fixture.GetComponentUnderTest<span class="hljs-tag">&lt;<span class="hljs-name">Heading</span>&gt;</span>();

      // Assert      
      // Here we get expected HTML from the Fragment element above.
      var expectedHtml = fixture.GetFragment();

      // Here we use the HTML diffing library to assert that the rendered HTML
      // from CUT is semantically the same as the expected HTML string above.
      cut.MarkupMatches(expectedHtml);
    }
  }
<span class="hljs-tag">&lt;/<span class="hljs-name">Fixture</span>&gt;</span>
</code>
```

And looks like this with the default highlight style:

![image](https://user-images.githubusercontent.com/105649/85617449-cc544300-b64e-11ea-8ef5-f43c37632a25.png)

Versions:

- Highlight.js 10.1.1 (93fd0d73)
- cshtml-razor (master branch from today) (b047bc1)

**Comment by romanresh** [comment]
Hi, @egil . The issue is fixed in v2.0.0.

**Comment by egil** [comment]
Thank you @romanresh! Much appreciated.
